book/installation.md

```diff
@@ -0,0 +1,38 @@
+# Build from Source
+
+You can build Ream on Linux.
+
+## Dependencies
+
+First install Rust using <a href="https://rustup.rs/">rustup</a>:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+There are some other dependencies you need to install based on your operating system (OS):
+
+- **Ubuntu/Debian**: `apt-get install libclang-dev pkg-config libssl-dev build-essential`
+
+
+
+## Build Ream
+
+Clone the repository and move to the directory:
+
+```bash
+git clone git@github.com:reamlabs/ream.git
+cd ream
+```
+
+There are also some other rust tools that you need to install.
+
+```bash
+cargo install cargo-sort
+```
```

Comment:
This is not necessary for building ream. We can put this under `Dependencies` for instance.